projects/packages/videopress/src/client/hooks/use-chapters-live-parsing/index.ts

```diff
@@ -0,0 +1,41 @@
+/**
+ * External dependencies
+ */
+import { useDebounce } from '@wordpress/compose';
+import { useCallback, useEffect, useState } from 'react';
+/**
+ * Internal dependencies
+ */
+import extractVideoChapters from '../../utils/video-chapters/extract-video-chapters';
+import validateChapters from '../../utils/video-chapters/validate-chapters';
+
+const CHAPTERS_CHECK_INTERVAL = 3000;
+
+const useChaptersLiveParsing = description => {
```

Comment:
minor suggestion: merge in case you need to re-run tests :-)

```suggestion
const useChaptersLiveParsing = ( description: string ) => {
```